### DISCUSSION TITLE
Remove guidelines overlay on rearview camera

### DISCUSSION THREAD

**Discussion by Compizfox** [opencardev/crankshaft#468] [Q&A] [answered]
Is it possible to remove the the guidelines that are added by cam_overlay to the rearview camera feed? My rearview camera already has an overlay like this in hardware, so right now I have *two* overlays.

**Comment by Jirradia** [comment, marked as answer]
Open your SD card in windows, navigate to crankshaft folder, replace camera-overlay.png with a clear image. done.

**Reply by Compizfox** [reply]
Thanks, I replaced `/opt/crankshaft/cam_overlay/overlay.png` with a transparent image of the same size.
